src/ci360.service.js

```diff
@@ -711,6 +731,21 @@ import {
     }
   }
 
+  showImageInfo(ctx) {
+    ctx.font = this.fullscreenView ? '28px serif' : '14px serif';
```

Comment:
maybe using template literals ``${this.fullscreenView ? '28px' : '14px'} serif``